notebooks/src/covid-19.ipynb

```diff
@@ -0,0 +1,798 @@
+{
```

Comment:
<p>If possible, I'd like to import <code>myqs</code>  and then just use <code>@myqs</code>  as the decorator, not <code>@myqs</code></p><p> </p><p>Not totally sure what's feasible though and not a big deal</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oughtinc/ergo/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>